pkgs/development/python-modules/fastmri/default.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+
+  # build system
+  setuptools,
+  setuptools-scm,
+
+  # dependencies
+  numpy,
+  scikit-image,
+  torchvision,
+  torch,
+  runstats,
+  pytorch-lightning,
+  h5py,
+  pyyaml,
+  torchmetrics,
+  pandas,
+
+  # tests
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "fastmri";
+  version = "0.3.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "fastMRI";
+    tag = "v${version}";
+    hash = "sha256-0IJV8OhY5kPWQwUYPKfmdI67TyYzDAPlwohdc0jWcV4=";
+  };
+
+  # banding_removal folder also has a subfolder named "fastmri"
+  # and np.product is substituted with np.prod in new numpy versions
+  postPatch = ''
+    substituteInPlace tests/test_math.py \
+      --replace-fail "np.product" "np.prod"
+    substituteInPlace tests/conftest.py \
+      --replace-fail "np.product" "np.prod"
```

Comment:
not a blocker

How about submitting the patch to the upstream?

Reply:
Tysm for the idea, i opened a PR but i dont know when itll be accepted.